packages/cozy-client/src/flagship-certification/flagship-certification.js

```diff
@@ -82,12 +82,19 @@ export const certifyFlagship = async (certificationConfig, client) => {
     )
   }
 
-  const stackChallengeNonce = await getStackChallenge(client)
+  try {
+    const stackChallengeNonce = await getStackChallenge(client)
 
-  const appAttestation = await getAppAttestationFromStore(
-    stackChallengeNonce,
-    certificationConfig
-  )
+    const appAttestation = await getAppAttestationFromStore(
+      stackChallengeNonce,
+      certificationConfig
+    )
 
-  await giveAppAttestationToStack(appAttestation, stackChallengeNonce, client)
+    await giveAppAttestationToStack(appAttestation, stackChallengeNonce, client)
+  } catch (e) {
+    console.warn(
+      '[FLAGSHIP_CERTIFICATION] Certification failed but the cozy-stack should continue with 2FA certification'
+    )
+    console.warn(e.message)
```

Comment:
should we use cozy-logger here 🙏 ?

Reply:
I don't remember, did we talked with @Crash-- about cozy-logger usage outside of konnectors?

It seems not to be used in cozy-client (expect for the CLI part) and using console.warn seems to be the common practice. Do we wan't to start using cozy-logger instead?

Reply:
I have no history on this.

Fow now, let's keep console warn and let's discuss about this with @Crash-- !

Reply:
https://github.com/cozy/cozy-client/blob/master/docs/logging.md

Reply:
So...  :  
```
import logger from './logger'
 logger.warn(``)
```

Reply:
I made the edit here: https://github.com/cozy/cozy-client/compare/d9ea74471e1e23ed356a9e710722a2939f4b6972..559190a18cb89ec0392b90d02f58b85f577e03e7